.github/workflows/build_packages.yml

```
@@ -36,9 +38,14 @@ jobs:
 
       - name: Build iree-turbine release candidate
         run: |
-          ./build_tools/compute_local_version.py -rc --write-json
+          version_name="$(./build_tools/compute_local_version.py -rc --write-json)"
```

Did you test this (on your fork)? Variables set like this are not persisted between workflow steps automatically.

See
* https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable
* https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-output-parameter


```suggestion
          version_name="$(./build_tools/compute_local_version.py -rc --write-json)"
          echo "version_name=${version_name}" >> "$GITHUB_ENV"
```

---

You are right.
I removed the tagging.

---

Ah, still not tested though? This failed last night:

https://github.com/iree-org/iree-turbine/actions/runs/13172140306/job/36764292832
```
usage: compute_local_version.py [-h] [--write-json]
                                (-stable | -rc | -dev | --version-suffix VERSION_SUFFIX)
compute_local_version.py: error: argument --version-suffix: not allowed with argument -rc/--nightly-release
```

I'd recommend testing on your fork using workflow dispatch, on https://github.com/sogartar/iree-turbine/actions/workflows/build_packages.yml

Hmm... looking through history, https://github.com/iree-org/iree/pull/19456 and https://github.com/iree-org/iree-turbine/pull/328 made the version suffix mutually exclusive with the other options. I had some discussions with @marbre about that... should have put more context in the PRs. Pasting here:

> when running the scripts locally, I certainly like the convenience of just doing -rc or -dev and having the script figure it out for me
> I don't think I'd ever want to do -rc --version-suffix
> if I'm specifying an explicit suffix, I just want that, no other stuff added for me

(oops, this is a good reason to want that combination of options 😅)